manifests/init.pp

```diff
@@ -283,171 +232,123 @@
-  $lazy_redirect_port        = $pulp::params::lazy_redirect_port,
-  $lazy_redirect_path        = $pulp::params::lazy_redirect_path,
-  $lazy_https_retrieval      = $pulp::params::lazy_https_retrieval,
-  $lazy_download_interval    = $pulp::params::lazy_download_interval,
-  $lazy_download_concurrency = $pulp::params::lazy_download_concurrency,
-  $consumers_crl             = $pulp::params::consumers_crl,
-  $reset_cache               = $pulp::params::reset_cache,
-  $ssl_verify_client         = $pulp::params::ssl_verify_client,
-  $ssl_protocol              = $pulp::params::ssl_protocol,
-  $repo_auth                 = $pulp::params::repo_auth,
-  $proxy_url                 = $pulp::params::proxy_url,
-  $proxy_port                = $pulp::params::proxy_port,
-  $proxy_username            = $pulp::params::proxy_username,
-  $proxy_password            = $pulp::params::proxy_password,
-  $yum_max_speed             = $pulp::params::yum_max_speed,
-  $num_workers               = $pulp::params::num_workers,
-  $enable_katello            = $pulp::params::enable_katello,
-  $enable_crane              = $pulp::params::enable_crane,
-  $max_tasks_per_child       = $pulp::params::max_tasks_per_child,
-  $enable_docker             = $pulp::params::enable_docker,
-  $enable_rpm                = $pulp::params::enable_rpm,
-  $enable_puppet             = $pulp::params::enable_puppet,
-  $enable_python             = $pulp::params::enable_python,
-  $enable_ostree             = $pulp::params::enable_ostree,
-  $enable_parent_node        = $pulp::params::enable_parent_node,
-  $enable_http               = $pulp::params::enable_http,
-  $manage_broker             = $pulp::params::manage_broker,
-  $manage_db                 = $pulp::params::manage_db,
-  $manage_httpd              = $pulp::params::manage_httpd,
-  $manage_plugins_httpd      = $pulp::params::manage_plugins_httpd,
-  $node_certificate          = $pulp::params::node_certificate,
-  $node_verify_ssl           = $pulp::params::node_verify_ssl,
-  $node_server_ca_cert       = $pulp::params::node_server_ca_cert,
-  $node_oauth_effective_user = $pulp::params::node_oauth_effective_user,
-  $node_oauth_key            = $pulp::params::node_oauth_key,
-  $node_oauth_secret         = $pulp::params::node_oauth_secret,
-  $disabled_authenticators   = $pulp::params::disabled_authenticators,
-  $additional_wsgi_scripts   = $pulp::params::additional_wsgi_scripts,
-  $puppet_wsgi_processes     = $pulp::params::puppet_wsgi_processes,
-  $migrate_db_timeout        = $pulp::params::migrate_db_timeout,
-  $show_conf_diff            = $pulp::params::show_conf_diff,
-  $enable_profiling          = $pulp::params::enable_profiling,
-  $profiling_directory       = $pulp::params::profiling_directory,
+  String $version = $::pulp::params::version,
+  Boolean $crane_debug = $::pulp::params::crane_debug,
+  Integer[0, 65535] $crane_port = $::pulp::params::crane_port,
```

Comment:
We use this so often, would it make sense to make a Port type, maybe in puppet-stdlib?